tests/dataset_tools/test_merge_sqfs_chunks.py

```
@@ -0,0 +1,482 @@
+        lines = [l for l in capsys.readouterr().out.strip().splitlines() if l]
+        names = [Path(l).name for l in lines]
+        assert names == ["chunk_0001.sqfs", "chunk_0002.sqfs", "chunk_0003.sqfs"]
+
+
+# ── main: streaming ───────────────────────────────────────────────────────────
+
+
+class TestStreaming:
+
+    def _run_stream(
+        self, staging: Path, extra_args: list[str] = []
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)